### ISSUE TITLE
Convert "Read about this artwork" into link

### ISSUE THREAD

**Issue by rumkin** [ethereum/ethereum-org-website#184]
Read about this artwork is a button, which behavs like a link. It should be replaced with proper element due to accessibility issues.

**Comment by samajammin** [comment]
Thanks for surfacing this. You're right, that callout is a button element but the button element is wrapped within an `<a>` link tag (with the URL in the `href`), so I believe it's fine from an accessibility standpoint.

Closing this out but feel free to re-open if you think there's something I'm missing.

**Comment by rumkin** [comment]
I see now. But I couldn't even think It could be done so. According to HTML standard interactive elements like buttons or inputs shouldn't be a child of an `<a>` element. Look at "Content Model" section and the last example block of Section [4.5.1 The a Element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element). Anyway it should be fixed and replaced with a `<span>` or other suitable element.

**Comment by samajammin** [comment]
Agreed - thanks for looking into that!
